### PR TITLE
Created contact form

### PR DESCRIPTION
**Pull request by leonard33**
Form creation and validation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,146 +1,275 @@
...
-        <link rel="preconnect" href="https://fonts.googleapis.com">
-        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
-        <link rel="preconnect" href="https://fonts.googleapis.com">
-        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
-        <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
-        <link rel="stylesheet" href="/style.css">
```
Comment (AmaduKamara) [inline index.html:25]: Your design seems broken as a result of the CSS link path

- [ ] Kindly adjust the path so the styles will take effect on the page
- [ ] Please use this instead `<link rel="stylesheet" href="./style.css">`. I placed a `"./style.css"` to the href and the design shows fine. Please fix that 👍 

image

Please take a look at the real design:
image

File: style.css [hunk 2]
```diff
@@ -478,3 +482,86 @@ ul li a {
...
+  font-size: 15px;
+}
+
+.submit-btn {
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
```
Comment (AmaduKamara) [inline style.css:548]: - [ ] Kindly implement interaction to the button on hover, focus to improve the good user experience of your app.
- 
image

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi, @leonard33  and @shaqdeff, 

Good job so far! 👍
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF (3)

## Highlights
✔️ Linters are working fine
✔️ Form is working fine
✔️ Nice commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -22,4 +22,5 @@ Show your support
 Give a ⭐️ if you like this project!
 
 Acknowledgment:
-More thanks for the knowledge on styling by using Figma
\ No newline at end of file
+More thanks for the knowledge on styling by using Figma
```
Comment (TedLivist) [inline README.md:25]: - [ ] **[OPTIONAL]** I would recommend you update your README with information regarding the project you are building now. You can use this [Microverse template](https://github.com/microverseinc/readme-template)

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @leonard33,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**